src/streamlink/plugins/telefe.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HLSStream
+
+
+class Telefe(Plugin):
+    _url_re = re.compile(r'https?://telefe.com/.+')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return Telefe._url_re.match(url)
+
+    def _get_streams(self):
+        TELEFE_UA = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36 OPR/44.0.2510.1218"
+        res = http.get(self.url, headers={'User-Agent': TELEFE_UA})
+        busqueda_videoID = res.content.decode()
+        busqueda_videoID = busqueda_videoID[busqueda_videoID.index('"PlayerContainer","model":{"id":') + 32 :]
+        busqueda_videoID = busqueda_videoID[: busqueda_videoID.index(',')]
+        #print('ID encontrado: ' + busqueda_videoID)
```

Comment:
You could make this line `self.logger.debug('ID encontrado: {0}', busqueda_videoID)` so that if there is a problem later we can get some debugging information :)